<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edycja_naprawa</class>
 <widget class="QDialog" name="edycja_naprawa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableView" name="tableView3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>511</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Załaduj tabelę</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DATAa_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>225</y>
     <width>133</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="s_data">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>199</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Data ostatniego serwisu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="s_opis">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>244</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Data ostatniego serwisu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_DATAa_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>133</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>281</y>
     <width>179</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj nowe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>339</y>
     <width>179</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>179</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edytuj...</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_idd">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>133</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
